src/dbnode/storage/cleanup.go

```diff
@@ -454,12 +454,10 @@ func (m *cleanupManager) cleanupSnapshotsAndCommitlogs() (finalErr error) {
 		// If we were unable to read the commit log files info header, then we're forced to assume
 		// that the file is corrupt and remove it. This can happen in situations where M3DB experiences
 		// sudden shutdown.
-		m.opts.InstrumentOptions().Logger().WithFields(
-			xlog.NewField("err", errorWithPath.Error()),
-			xlog.NewField("path", errorWithPath.Path()),
-		).Errorf(
-			"encountered corrupt commitlog file during cleanup, marking file for deletion: %s",
-			errorWithPath.Error())
+		m.opts.InstrumentOptions().Logger().With(
+			zap.Error(errorWithPath),
```

Comment:
Same here with `zap.Error()` at the end.

Reply:
done